test/source/tests/tests/decrypt.ts

```diff
@@ -191,6 +191,13 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithNewBrowser:
       await InboxPageRecipe.checkDecryptMsg(t, browser, { acctEmail, threadId, expectedContent });
     }));
 
+    ava.default('[standalone] - inbox - encrypted message inside signed', testWithSemaphoredBrowser('compatibility', async (t, browser) => {
+      const inboxPage = await browser.newPage(t, 'chrome/settings/inbox/inbox.htm?acctEmail=flowcrypt.compatibility%40gmail.com&threadId=16f0bfce331ca2fd');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
```

Comment:
You have to first await the frame above, look at other usages of `getFrame`

Reply:
That will make such tests a little lest flaky, I think

Reply:
Agree, Thanks!